lib/presentation/widgets/word_cloud.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_scatter/flutter_scatter.dart';
+
+import '../../core/theme/app.dart';
+import '../models/word_cloud/word_cloud_data.dart';
+
+class WordCloudExample extends StatelessWidget {
+  const WordCloudExample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    List<Widget> widgets = <Widget>[];
+    for (var i = 0; i < kFlutterHashtags.length; i++) {
+      widgets.add(ScatterItem(kFlutterHashtags[i], i));
+    }
+
+    final screenSize = MediaQuery.of(context).size;
+    final ratio = screenSize.width / screenSize.height;
+
+    return SizedBox(
+      height: AppHeightValues.sentimentChartHeight,
+      child: Padding(
+        padding: const EdgeInsets.all(AppPaddingValues.xSmallHorizontalPadding),
+        child: Card(
+          child: Center(
+            child: Padding(
+              padding: const EdgeInsets.all(AppPaddingValues.smallPadding),
+              child: FittedBox(
+                child: Scatter(
+                  fillGaps: true,
+                  delegate: ArchimedeanSpiralScatterDelegate(ratio: ratio),
+                  children: widgets,
```

Comment:
call `_generateScatterItems(data)` method here

Reply:
Addressed in commit 93da88eb9e46d98805eb2098d68d0950c35626fe